src/chainstate/burn/db/sortdb.rs

```diff
@@ -470,22 +471,19 @@ const BURNDB_SETUP: &'static [&'static str] = &[
 
         PRIMARY KEY(sortition_id)
     );"#,
-    r#"
-    CREATE UNIQUE INDEX snapshots_block_hashes ON snapshots(block_height,index_root,winning_stacks_block_hash);
-    CREATE UNIQUE INDEX snapshots_block_stacks_hashes ON snapshots(num_sortitions,index_root,winning_stacks_block_hash);
-    CREATE UNIQUE INDEX snapshots_block_heights ON snapshots(burn_header_hash,block_height);
-    CREATE UNIQUE INDEX snapshots_burn_hashes ON snapshots(block_height,burn_header_hash)
-    CREATE INDEX snapshots_block_winning_hash ON snapshots(winning_stacks_block_hash);
-    CREATE INDEX block_arrivals ON snapshots(arrival_index,burn_header_hash);
-    CREATE INDEX arrival_indexes ON snapshots(arrival_index);
-    "#,
+    "CREATE UNIQUE INDEX snapshots_block_hashes ON snapshots(block_height,index_root,winning_stacks_block_hash);",
+    "CREATE UNIQUE INDEX snapshots_block_stacks_hashes ON snapshots(num_sortitions,index_root,winning_stacks_block_hash);",
+    "CREATE UNIQUE INDEX snapshots_block_heights ON snapshots(burn_header_hash,block_height);",
+    "CREATE UNIQUE INDEX snapshots_burn_hashes ON snapshots(block_height,burn_header_hash);",
+    "CREATE INDEX snapshots_block_winning_hash ON snapshots(winning_stacks_block_hash);",
+    "CREATE INDEX block_arrivals ON snapshots(arrival_index,burn_header_hash);",
+    "CREATE INDEX arrival_indexes ON snapshots(arrival_index);",
```

Comment:
Instead of turning each statement into its own string, why not use `execute_batch()`?

Reply:
Yeah I don't have a strong opinion. Executing statement per statement is probably easier to debug?

Reply:
This PR was about patching the issue, in my mind we need to revisit this strategy very soon with the ability of recovering a `working_dir`

Reply:
Ideally, we execute the statements one by one, ensure that one statement contains only one statement, log the SQL statements failing, etc. This granularity would be hard to achieve with one big blob of SQL statements.

Reply:
> Yeah I don't have a strong opinion. Executing statement per statement is probably easier to debug?

The problem was that adding additional statements into the same string was causing them to silently be ignored.  I think that the fix to this problem is to make it so that all statements get executed, no matter how they are represented in the code (either as separate strings, or grouped together into a single string).  You might argue that putting multiple statements into the same string is in and of itself a bug, but continuing with this path sets us up for future failure -- the compiler won't warn us when we do this, nor will the DB instantiation logic, and we'll have this problem again.

Also, the this PR's diff would be _smaller_ if we used `execute_batch()`.

Reply:
To be clear, I don't have an opinion on whether or not the PR should split the SQL schemas into one statement per string.  However, I have a very _strong_ opinion that we should use `execute_batch()`, since that would have prevented this problem in the first place.

Reply:
Sounds good, addressed with https://github.com/blockstack/stacks-blockchain/pull/2420/commits/90b1bceb59313d54ce1a3a35a0361ce9f30ca448